src/compile/unit.ts

```diff
@@ -72,31 +71,21 @@ export class UnitModel extends ModelWithField {
     parent: Model,
     parentGivenName: string,
     parentGivenSize: LayoutSizeMixins = {},
-    repeater: RepeaterValue,
     config: Config
   ) {
-    super(
-      spec,
-      'unit',
-      parent,
-      parentGivenName,
-      config,
-      repeater,
-      undefined,
-      isFrameMixins(spec) ? spec.view : undefined
-    );
+    super(spec, 'unit', parent, parentGivenName, config, undefined, isFrameMixins(spec) ? spec.view : undefined);
 
     const mark = isMarkDef(spec.mark) ? spec.mark.type : spec.mark;
 
-    const encodingWithRepeaterReplaced = replaceRepeaterInEncoding(spec.encoding ?? {}, repeater);
+    const encoding = spec.encoding ?? {};
 
-    this.markDef = initMarkdef(spec.mark, encodingWithRepeaterReplaced, config, {
+    this.markDef = initMarkdef(spec.mark, encoding, config, {
       graticule: spec.data && isGraticuleGenerator(spec.data)
     });
-    const encoding = (this.encoding = initEncoding(encodingWithRepeaterReplaced, this.markDef));
+    const normalizedEncoding = (this.encoding = initEncoding(encoding, this.markDef));
```

Comment:
Let's not call this normalized as it doesn't happen in the normalized phase. Instead, just call this encoding and refer to the old `encoding` above via `spec.encoding`.

Reply:
Done